docs/flashbots-mev-share/introduction.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Introduction
+---
+
+import ProtectButtonSelector from "@site/src/components/ProtectButtonSelector"
+import WhatsMevShareAdvanced from "../specs/mev-share/blurbs/_whatsMevShareAdvanced.mdx"
+
+> The MEV-Share Matchmaker was renamed to the MEV-Share Node to better reflect the role we envision this actor will play in SUAVE. This change will be reflected in our documentation after June 2023. The client library previously named `matchmaker-ts` has also been renamed to `mev-share-client-ts`.
+
+## What is MEV-Share?
+<WhatsMevShareAdvanced />
+
+To start earning MEV refunds from MEV-share, submit your transactions to the Flashbots **MEV-Share Node**, either by [connecting your wallet to the Protect RPC](#connect-your-wallet), or sending a [private transaction](/flashbots-auction/searchers/advanced/rpc-endpoint#eth_sendprivatetransaction) to the MEV-Share Node API.
+
+## Why MEV-Share?
+MEV-Share redistributes MEV back to the party that creates it in the first place: users. It does so through a design that is built to scale and be decentralized, which means that it is permissionless for searchers to integrate and it does not enshrine a single block builder.
+
+## How does MEV-Share work?
```

Comment:
Did we remove the line about only backruns being supported on purpose?

Reply:
This info is also quite important -- not sure if it was moved elsewhere but it should not be deleted.

> Currently, MEV-Share only allows constructing bundles containing a single input and a single backrun. If the input is also a bundle, the rule applies recursively to that nested bundle. Concretely this means that searchers can only place hashes in the first position of their bundles; and each bundle may include up to one hash. The MEV-Share Node will reject any bundles that violate these rules. This ensures that searchers cannot frontrun users in-protocol.